connector-packager/connector_packager/jar_jdk_packager.py

```diff
@@ -44,6 +44,10 @@ def get_min_support_version(file_list: List[ConnectorFile]) -> str:
             min_version_tableau = "2020.3"
             reasons.append("Connector uses Connection Dialogs V2, which was added in the 2020.3 release")
 
+    if float(cur_min_version_tableau) > float(min_version_tableau):
```

Comment:
[The XSD schema allows the version be more than two parts](https://github.com/tableau/connector-plugin-sdk/blob/master/validation/connector_plugin_manifest_latest.xsd#L92). For example, it could be 2020.4.2. 

We can't use float to compare versions.

Reply:
Good catch.